src/index.html

```diff
@@ -10,13 +10,34 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="Logo"/>
+      </a>
+      <nav>
+        <ul class="menu">
+          <li><a href="#index.html" class="is-active">APPLE</a></li>
```

Comment:
Fix classes and code style everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#index.html" class="nav__link is-active">APPLE</a>
          </li>
```